contract-map.json

```diff
@@ -20,6 +20,13 @@
     "symbol": "CHAI",
     "decimals": 18
   },
+  "0x04fa0d235c4abf4bcf4787af4cf447de572ef828": {
```

Comment:
Please update your address string to the following checksum address:
```suggestion
  "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828": {
```
In the future, you can obtain a checksum address using `ethereumjs-util` (`toChecksumAddress`).